### DISCUSSION TITLE
White screen while using vue-flow-form on my website

### DISCUSSION THREAD

**Discussion by AriaAman** [ditdot-dev/vue-flow-form#246] [Q&A] [answered]
Hi guys, 
I hope you are doing well,

I'm new to Vue.js and I wanted to add the vue-flow-form on my website but when i do exactly as it says in the git guide, it just gives me a white screen :/ 

my vue version is 3.1.5,
and my npm is above version 5
![image](https://user-images.githubusercontent.com/75366230/168077111-d7726ab0-215e-4321-8396-5cdecae0be9a.png)

this is the code on my "app.vue" file :
![image](https://user-images.githubusercontent.com/75366230/168077519-59adcf88-2ba7-4a33-a797-6dce02849f2e.png)
![image](https://user-images.githubusercontent.com/75366230/168077547-892135c8-3385-4bd2-a094-703744d6de43.png)

before adding the the vue-form:
![image](https://user-images.githubusercontent.com/75366230/168076608-9436879c-f24f-44b7-bd7b-4f1b3e9906f9.png)

after adding the vue-form: 
![image](https://user-images.githubusercontent.com/75366230/168076401-ecc77eee-ccaa-47cc-9335-530dac2a408e.png)

Thank you in advance to anyone who is helping me :D
I wish you a good day !

**Comment by spinn** [comment, marked as answer]
Hi @AriaAman,

you're using Vue Router so our example will not work without modifications in your use case. You'll need to set-up the router to load a component containing Vue Flow Form in a route of your choice. More information can be found in Vue Router's guide:

https://router.vuejs.org/guide

**Comment by McFlyin619** [comment]
in main.js you can import them as well
`
import '../node_modules/@ditdot-dev/vue-flow-form/dist/vue-flow-form.css'

import '../node_modules/@ditdot-dev/vue-flow-form/dist/vue-flow-form.theme-purple.css'
`
